packages/components/tabs/src/tab-pane.vue

```diff
@@ -2,15 +2,16 @@
   <div
     v-if="shouldBeRender"
     v-show="active"
-    :id="`pane-${paneName}`"
+    :id="`pane-${isNumber(paneName) ? 'n-' : ''}${paneName}`"
     :class="ns.b()"
     role="tabpanel"
     :aria-hidden="!active"
-    :aria-labelledby="`tab-${paneName}`"
+    :aria-labelledby="`tab-${isNumber(paneName) ? 'n-' : ''}${paneName}`"
```

Comment:
I found that there are many similar codes, can we encapsulate it into a general function?
like this:
```ts 
const genId = (type: 'tab' | 'pane', name: string) => {
  return `${type}-${isNumber(name) ? 'n-' : ''}${name}`
}
```

Reply:
@buqiyuan I refactored the relevant code

Reply:
/cc @element-plus/backers  Should we be compatible with string and number types

Reply:
> Should we be compatible with string and number types

In my opinion, they should be string at last, right?

Reply:
Exactly so, on [line 44](https://github.com/element-plus/element-plus/blob/dev/packages/components/tabs/src/tab-bar.vue#L44) will not be able to distinguish they.

Reply:
Do we need to distinguish between them?

Reply:
0 -> 'tab-0'
'0' -> 'tab-0'
The id and key need to be unique values, so there are some potential risks.

Of course, it is also possible not to distinguish between them. This issues can also be fixed by class name, but I am not sure if it is a good idea.